lexers/embedded/hare.xml

```diff
@@ -0,0 +1,101 @@
+<lexer>
+  <config>
+    <name>Hare</name>
+    <alias>hare</alias>
+    <filename>*.ha</filename>
+    <mime_type>text/x-hare</mime_type>
+  </config>
+  <rules>
+    <state name="whitespace">
+      <rule pattern="[\s\n]+">
+        <token type="TextWhitespace"/>
+      </rule>
+      <rule pattern="@[a-z]+">
+        <token type="NameDecorator"/>
```

Comment:
As far as I know attributes (as Hare calls `@test` and friends) are compiler directives, so should I use `CommentPreproc` here instead?

Reply:
I think NamDecorator is probably more like what other languages use.